### DISCUSSION TITLE
On Updating Kitty

### DISCUSSION THREAD

**Discussion by ad-chaos** [kovidgoyal/kitty#5648] [General]
The way I update kitty is by simply running the `curl -L https://sw.kovidgoyal.net/kitty/installer.sh | sh /dev/stdin` command whenever a new release shows up. Though the problem with this approach is that I have to give permissions to the newly installed kitty to my `Documents`, `Downloads` and other folders that mac restricts access to, which is slightly annoying.

People who use kitty on macos, how to do you get around this? I did a little bit of looking on the internet but couldn't find a nice solution other than completely disabling that feature :(

**Comment by tg0h** [comment]
Hmm strangely enough I don't have this problem.

I installed kitty via homebrew - https://formulae.brew.sh/cask/kitty

I have a brew file with all the stuff I use
<img width="621" alt="image" src="https://user-images.githubusercontent.com/7954426/200431883-aa6d77d1-51af-4499-a271-db5983116a44.png">

I then run `brew bundle --file ~/dotfiles/config/homebrew/brewfile` periodically to update all my stuff

If I just want to update any 1 particular tool, I can run `brew upgrade kitty` eg

**Reply by ad-chaos** [reply]
But is the kitty cask maintained? I thought the only official binaries were the ones present in the releases page and subsequently the one that the `curl` command downloads?

**Comment by tg0h** [comment]
Yes, it's maintained. Just got the latest release.

<img width="821" alt="image" src="https://user-images.githubusercontent.com/7954426/200695541-b9ad46a9-e488-47d6-b881-815f055ff415.png">

**Reply by ad-chaos** [reply]
Alright thanks, will give that a try from the next release onwards :)
